e2e-tests/test-utils/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "test-utils"
+version = "0.5.1-devnet5"
+edition = "2024"
+description = "Test utilities for XRPL WASM e2e tests"
```

Comment:
Perhaps there should be some note here that this is a private package and won't/shouldn't be published to crates.io